examples/with-solana/src/addSignature.ts

```diff
@@ -56,22 +56,22 @@ async function main() {
         `\n--------`,
       ].join("\n")
     );
-    await confirm({ message: "Ready to Continue?" });
+    await prompts({ message: "Ready to Continue?" });
     // refresh balance...
     balance = await solanaNetwork.balance(connection, solAddress);
   }
 
   print("SOL balance:", `${balance} Lamports`);
 
   // 1. Create, sign, and verify a transfer transaction using `addSignature` as opposed to `signTransaction`
-  const destination = await input({
+  const destination = await prompts({
```

Comment:
was this tested? there are some syntax differences with using `prompts` vs. `inquirer/prompts`. you can refer to the stackblitz example itself https://stackblitz.com/edit/stackblitz-starters-xeb93i?file=src%2Findex.ts